build/common.sh

```diff
@@ -107,28 +107,35 @@ readonly KUBE_APISERVER_BASE_IMAGE="${KUBE_APISERVER_BASE_IMAGE:-$KUBE_GORUNNER_
 readonly KUBE_CONTROLLER_MANAGER_BASE_IMAGE="${KUBE_CONTROLLER_MANAGER_BASE_IMAGE:-$KUBE_GORUNNER_IMAGE}"
 readonly KUBE_SCHEDULER_BASE_IMAGE="${KUBE_SCHEDULER_BASE_IMAGE:-$KUBE_GORUNNER_IMAGE}"
 readonly KUBE_PROXY_BASE_IMAGE="${KUBE_PROXY_BASE_IMAGE:-$KUBE_BASE_IMAGE_REGISTRY/distroless-iptables:$__default_distroless_iptables_version}"
+readonly KUBE_PROXY_WINDOWS_BASE_IMAGE="${KUBE_PROXY_WINDOWS_BASE_IMAGE:-mcr.microsoft.com/oss/kubernetes/windows-host-process-containers-base-image:v1.0.0}"
```

Comment:
We don't currently depend directly on any third party endpoints for building releases, so this is a regression.

Reply:
The comparable base images for linux have sources in kubernetes/release and then are promoted from there and used by this and other repos. I think that might be the best approach?

Licensing is important, but I was actually bringing it up from the runtime dependency angle of "your source code won't build unless this host or a mirror is up", right now we have that down to github + registry.k8s.io for the core repo, more or less (not even the go module proxy, we check in vendor here).

If you're willing to move the sources, that's even better, though sig release has to accept or maybe we find a sig windows repo.

cc @kubernetes/release-engineering

Reply:
I can move the sources into k/release.
Let me try and find someone who can verify that is OK from sig release.